sktime/forecasting/online_ensemble.py

```diff
@@ -0,0 +1,171 @@
+import numpy as np
+import pandas as pd
+from sktime.forecasting.base._base import DEFAULT_ALPHA
+from sktime.forecasting.compose._ensemble import EnsembleForecaster
+from sktime.forecasting.model_selection import OnlineSplitter
+from .ensemble_algorithms import EnsembleAlgorithms
+
+
+class OnlineEnsembleForecaster(EnsembleForecaster):
+    """Online Updating Ensemble of forecasters
+
+    Parameters
+    ----------
+    ensemble_algorithm : ensemble algorithm
+    forecasters : list of (str, estimator) tuples
+    n_jobs : int or None, optional (default=None)
+        The number of jobs to run in parallel for fit. None means 1 unless
+        in a joblib.parallel_backend context.
+        -1 means using all processors.
+    """
+
+    _required_parameters = ["forecasters"]
+
+    def __init__(self, forecasters, ensemble_algorithm=None, n_jobs=None):
+        self.n_jobs = n_jobs
+        if ensemble_algorithm is None:
+            self.ensemble_algorithm = EnsembleAlgorithms(len(forecasters))
+        else:
+            self.ensemble_algorithm = ensemble_algorithm
+
+#         if self.ensemble_algorithm.n != len(forecasters):
+#             raise ValueError("Number of Experts in Ensemble Algorithm \
+#                              doesn't equal number of Forecasters")
+
+        super(EnsembleForecaster, self).__init__(forecasters=forecasters,
+                                                 n_jobs=n_jobs)
+
+    def _fit_ensemble(self, y_val, X_val=None):
```

Comment:
I'd prefer `y_new` and `X_new` to make it consistent with the rest here and throughout the code